src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Fix `System.FormatException` thrown by `PersistentStorageEventSource`.
```

Comment:
No changelog is needed here. This component is not yet used by any code path. The changelog needs to be added [here](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.PersistentStorage.FileSystem)